src/card.ts

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+} from './'
+
+export interface ExpirationOffset {
+  unit?: string;
+  value?: bigint;
+}
+
+export interface AddressType {
+  firstName?: string;
+  middleName?: string;
+  lastName?: string;
+  address1: string;
+  address2?: string;
+  city?: string;
+  state?: string;
+  zip?: string;
+  country?: string;
+  phone?: string;
+  postalCode?: string;
+}
+
+export interface Shipping {
+  method?: string;
+  returnAddress?: AddressType;
+  recipientAddress?: AddressType;
+  carOfLine?: string;
```

Comment:
Is this really `carOfLine`? I think maybe `careOfLine` - Right?

Reply:
typo: it is `careOfLine`